packages/atomic/vitest.config.ts

```diff
@@ -37,6 +37,7 @@ export default defineConfig({
     alias: {
       '@/': path.resolve(import.meta.dirname, './') + '/',
     },
+    conditions: ['coveoInternalCDN'],
```

Comment:
This allows to circumvent the many `export *` present within `@coveo/headless` that `vitest` cannot digest easily in browser mode.